css/sidebar.css

```diff
@@ -38,51 +38,51 @@
     background:#3e3b44;
     padding:17px 000;
 }
-.social-icon{
+.sidebar-social-icon{
     width:99.99%;
     height:100%;
```

Comment:
po co? Jeśli to nie jest position absolute/fixed - to height 100% nic Ci nie da. 
Usuń.